lib/Dialect/TritonGPU/Transforms/Pipeliner/SoftwarePipeliner.cpp

```diff
@@ -52,8 +52,10 @@ static bool preCondition(scf::ForOp forOp) {
               return WalkResult::interrupt();
             return WalkResult::advance();
           })
-          .wasInterrupted())
+          .wasInterrupted()) {
+    forOp->emitRemark() << "Warning: SWP fails on the outer loop";
```

Comment:
that might be a bit of a noisy comment as pipelining is known to be an inner loop transformation. Do we really need to tell user that the outer loop didn't pipeline?

Reply:
@ThomasRaoux, the PR is updated with your comments resolved, could you take a look? Thanks